shared/helpers.ts

```diff
@@ -1,211 +1,170 @@
-
-        if (platform === 'darwin') {
-            osAndArch = 'mac';
-        }
-        if (platform === 'linux') {
-            if (arch === 'x64') {
-                osAndArch = 'linux-x86_64';
-            }
-            if (arch === 'x32') {
-                osAndArch = 'linux-i686';
-            }
-        }
-        if (platform === 'win32') {
-            if (arch === 'x64') {
-                osAndArch = 'win64';
-            }
-            if (arch === 'x32') {
-                osAndArch = 'win32';
-            }
-        }
-
-        return osAndArch;
-    }
-
-    getPlatform() {
-        global.platform = {
-            darwin: platform === 'darwin',
-            linux: platform === 'linux',
-            win32: platform === 'win32'
-        }
-    }
-
-    getHTTPorHTTPs() {
-        if (global.platform.win32) {
-            return https;
-        }
-        return http;
-    }
-
-    fixPath(pathStr: string) {
-        if (global.platform.win32) {
-            return pathStr.split(path.sep).join(path.posix.sep);
-        }
-        // linux & mac
-        return pathStr;
-    }
-
-    async getPNPath(osAndArch: any) {
-        // const definitelyPosix = projectDir.split(path.sep).join(path.posix.sep);
-        return path.join(os.homedir(), '.point', 'src', 'pointnetwork');
-    }
-
-    async getDashboardPath(osAndArch: any) {
-        return path.join(os.homedir(), '.point', 'src', 'pointnetwork-dashboard');
-    }
-
-    async getSDKPath(osAndArch: any) {
-        return path.join(os.homedir(), '.point', 'src', 'pointsdk');
-    }
+import { app } from 'electron'
+import { http, https } from 'follow-redirects'
+import path from 'path'
+import fs from 'fs-extra'
+import os from 'os'
+import { platform, arch } from 'process'
+
+const getOSAndArch = () => {
+  console.log('platform', platform)
+  // Returned values: mac, linux-x86_64, linux-i686, win64, win32, or throws an error
+  let osAndArch = ''
+
+  if (platform === 'darwin') {
+    osAndArch = 'mac'
+  }
+  if (platform === 'linux') {
+    if (arch === 'x64') {
+      osAndArch = 'linux-x86_64'
+    }
+    if (arch === 'x32') {
+      osAndArch = 'linux-i686'
+    }
+  }
+  if (platform === 'win32') {
+    if (arch === 'x64') {
+      osAndArch = 'win64'
+    }
+    if (arch === 'x32') {
+      osAndArch = 'win32'
+    }
+  }
+
+  return osAndArch
+}
 
-    async getBrowserFolderPath(osAndArch: any) {
-        // const definitelyPosix = projectDir.split(path.sep).join(path.posix.sep);
-        const browserDir = path.join(os.homedir(), '.point', 'src', 'point-browser');
-        if (!fs.existsSync(browserDir)) {
-            fs.mkdirpSync(browserDir);
-        }
-        return browserDir;
-    }
+const getPlatform = () => {
+  global.platform = {
+    darwin: platform === 'darwin',
+    linux: platform === 'linux',
+    win32: platform === 'win32',
+  }
+}
 
-    getHomePath() {
-        // TODO: Delete if `os.homedir` works for Windows, too (because of WSL).
```

Comment:
The `os.homedir()` function works perfectly fine for windows so I've removed some commented out logic.